Userland/Shell/main.cpp

```diff
@@ -169,13 +170,15 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     char const* format = nullptr;
     bool should_format_live = false;
     bool keep_open = false;
+    bool posix_mode = false;
 
     Core::ArgsParser parser;
     parser.add_option(command_to_run, "String to read commands from", "command-string", 'c', "command-string");
     parser.add_option(skip_rc_files, "Skip running shellrc files", "skip-shellrc", 0);
     parser.add_option(format, "Format the given file into stdout and exit", "format", 0, "file");
     parser.add_option(should_format_live, "Enable live formatting", "live-formatting", 'f');
     parser.add_option(keep_open, "Keep the shell open after running the specified command or file", "keep-open", 0);
+    parser.add_option(posix_mode, "Pretend this is a good law-abiding POSIX /bin/sh", "posix", 0);
```

Comment:
No jokes in the system, "Make Shell behave as a POSIX /bin/sh" (or similar) is easier to understand.

Reply:
But film, POSIX _is_ a joke
Is that even a joke? I'll change it to "Behave as a POSIX-compatible shell" :shrug: